addons/website_sale/views/sale_order_views.xml

```diff
@@ -7,10 +7,33 @@
         <field name="arch" type="xml">
             <search>
                 <filter string="Confirmed Orders" name="order_confirmed" domain="[('state', 'in', ('sale', 'done'))]"/>
+                <separator/>
+                <filter string="From Website" name="from_website" domain="[('team_id.team_type', '=', 'website')]"/>
+                <separator/>
+                <filter string="Recovery Email to Send" name="recovery_email" domain="[('cart_recovery_email_sent', '=', False)]" invisible="1"/>
             </search>
         </field>
     </record>
 
+    <record id="view_order_form_inherit_website_sale" model="ir.ui.view">
```

Comment:
JustNamingThings: we have some guidelines on how to write xml_ids. For example for a view
```
<model>_view_<view_type>_<suffix_for_details>
```
It could be sale_order_view_form_inherit_cart_recovery for example.